crates/matrix-sdk-ui/src/timeline/date_dividers.rs

```diff
@@ -192,8 +192,16 @@ impl DateDividerAdjuster {
 
         // Only record the initial state if we've enabled the trace log level, and not
         // otherwise.
-        let initial_state =
-            if event_enabled!(Level::TRACE) { Some(items.iter().cloned().collect()) } else { None };
+        let initial_state = if event_enabled!(Level::TRACE) {
+            Some(
+                items
+                    .iter_remotes_and_locals_regions()
```

Comment:
For recording the initial state, don't we want all regions?

Reply:
It's the initial state for the date divider: it doesn't need all the regions; only the remotes and locals regions!